src/main/java/mcmultipart/api/multipart/IMultipart.java

```diff
@@ -281,6 +281,14 @@ public default void onAdded(IPartInfo part) {
     public default void onRemoved(IPartInfo part) {
     }
 
+    public default boolean canPlacePartAt(IBlockAccess world, BlockPos pos) {
+        return true;
```

Comment:
No, no, no! These should be proxying to the getBlock()'s original methods! But that's tricky, as the IPartInfo doesn't exist yet. I was writing a request to @amadornes to clarify what to do here, but you pushed this quicker... ah well.

Reply:
hmm... I called IMultipart.getBlock() in the placement code, I imagine it's fine here too? But good point, they should query the Block methods. Whoops :P

Reply:
You *are* in the IMultipart, you can probably just call getBlock().

It may not be the finest way, but it's sufficient to proxy everything vanilla is asking of you.

Reply:
Yeah, it seems to be working fine calling getBlock().canPlaceBlockAt(…)